src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -230,6 +240,39 @@ class ProductaanvraagService @Inject constructor(
         return initiatorBetrokkene
     }
 
+    private fun Betrokkene.shouldBeSkipped(
+        zaak: Zaak,
+        genericRole: Boolean,
+        brpEnabled: Boolean,
+        kvkEnabled: Boolean
+    ): String? {
+        val rolTypeDescription = this.roltypeOmschrijving ?: this.rolOmschrijvingGeneriek.toString()
+        val prefix = if (genericRole) "generic " else ""
+
+        return this.performAction(
+            onNatuurlijkPersoonIdentity = {
+                if (!brpEnabled) {
+                    "Betrokkene with ${prefix}roletype description `$rolTypeDescription` has BRP-based identity, but BRP " +
+                        "is not enabled for zaak type ${zaak.zaaktype}. No betrokkene role created for zaak ${zaak.identificatie}"
```

Comment:
`${zaak.zaaktype}` will print a huge amount of information I think? Better just to print the zaaktype description? That is all what matters in this context.

Reply:
That's just the URI, so not that big. We don't have zaaktype object here and I'm a bit hesitant to propagate it down the call stack. 

We log zaaktype URI and zaak id, so that should be enough to be able to debug/reproduce.